x-pack/plugins/monitoring/public/components/elasticsearch/shard_allocation/components/cluster_view_react.js

```diff
@@ -8,21 +8,20 @@
 import React from 'react';
 import { TableHeadReact } from './table_head_react';
 import { TableBody } from './table_body';
-import { labels } from '../lib/labels';
 
 export const ClusterViewReact = (props) => {
   return (
     <table cellPadding="0" cellSpacing="0" className="table">
       <TableHeadReact
-        labels={labels.node}
+        labels={props.labels}
```

Comment:
this has to be sent from the parent component (index or node pages) because they both use this component and pass in their respective labels